src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1502,23 +1502,30 @@ lemma preconnected.subsingleton_connected_component (h : G.preconnected) :
 ⟨connected_component.ind₂ (λ v w, connected_component.sound (h v w))⟩
 
 /-- The map on connected components induced by a graph homomorphism. -/
-def connected_component.map {V : Type*} {G : simple_graph V} {V' : Type*} {G' : simple_graph V'}
-  (φ : G →g G') (C : G.connected_component) : G'.connected_component :=
+def connected_component.map  (φ : G →g G') (C : G.connected_component) : G'.connected_component :=
 C.lift (λ v, G'.connected_component_mk (φ v)) $ λ v w p _,
   connected_component.eq.mpr (p.map φ).reachable
 
-@[simp] lemma connected_component.map_mk
-  {V : Type*} {G : simple_graph V} {V' : Type*} {G' : simple_graph V'} (φ : G →g G') (v : V) :
+@[simp] lemma connected_component.map_mk (φ : G →g G') (v : V) :
   (G.connected_component_mk v).map φ = G'.connected_component_mk (φ v) := rfl
 
 @[simp] lemma connected_component.map_id (C : connected_component G) : C.map hom.id = C :=
 by { refine C.ind _, exact (λ _, rfl) }
 
-@[simp] lemma connected_component.map_comp
-  {V' : Type*} {G' : simple_graph V'} {V'' : Type*} {G'' : simple_graph V''}
-  (C : G.connected_component) (φ : G →g G') (ψ : G' →g G'') : (C.map φ).map ψ = C.map (ψ.comp φ) :=
+@[simp] lemma connected_component.map_comp (C : G.connected_component)
+  (φ : G →g G') (ψ : G' →g G'') : (C.map φ).map ψ = C.map (ψ.comp φ) :=
 by { refine C.ind _, exact (λ _, rfl), }
 
+/-- An isomorphism of graphs induces a bijection of connected components. -/
+@[simps]
+def connected_component.iso (φ : G ≃g G') : G.connected_component ≃ G'.connected_component :=
```

Comment:
Can you add the trivial lemmas that
* `connected_component.iso refl = refl`
* `(connected_component.iso e).symm = connected_component.iso e.symm`
* `(connected_component.iso e).trans (connected_component.iso f) = connected_component.iso (e.trans f)`

I don't remember which way around is preferred for the last two, but they're good to have one way or the other.

Reply:
The preferred ways are:
* `connected_component.iso e.symm = (connected_component.iso e).symm`
* `connected_component.iso (e.trans f) = (connected_component.iso e).trans (connected_component.iso f)`